src/store/index.ts

```diff
@@ -1,82 +1,28 @@
-import { createStore, applyMiddleware } from 'redux';
+import { createStore, applyMiddleware, combineReducers } from 'redux';
 import { composeWithDevTools } from 'redux-devtools-extension';
 import thunk from 'redux-thunk';
-
-type StartLoadingAction = {
-  // we use a literal as a type so the type can't have any other value
-  type: 'START_LOADING',
-};
-
-type FinishLoadingAction = {
-  type: 'FINISH_LOADING',
-  payload: string,
-};
-
-// Only the listed actions can be dispatched in the App
-type Action = (
-  StartLoadingAction
-  | FinishLoadingAction
-);
+import { Todo } from '../types/Todo';
+import { selectedTodoReducer } from './сurrentTodo';
+import loadingReducer from './loading';
 
 type RootState = {
```

Comment:
You do not need to define RootState type by yourself

Use this
`export type RootState = ReturnType<typeof rootReducer>;`